cmd/resources/resources.go

```diff
@@ -283,6 +283,7 @@ func (op *OperationCmd) initFlags() error {
 }
 
 func buildURLWithParams(baseURI, path string, urlParams []string) string {
```

Comment:
There aren't tests for this, and I didn't want to possibly break things, so this will remove the trailing slash or be a no-op.